website/docs/docs/dbt-versions/release-notes/09-April-2023/api-endpoint-restriction.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "List Runs API Endpoint order_by restrictions"
+id: api-endpoint-restriction
+sidebar_label: "Deprecation: "List Runs API Endpoint order_by restrictions"
```

Comment:
```suggestion
sidebar_label: "Deprecation: "List Runs API Endpoint `order_by` restrictions"
```

Reply:
@mirnawong1 It might be failing because of this in the sidebar title. Removing and seeing if that helps